core/ui/EditTemplate/fields.tid

```diff
@@ -69,7 +69,7 @@ $value={{{ [<newFieldValueTiddler>get[text]] }}}/>
 <$fieldmangler>
 <div class="tc-edit-field-add">
 <em class="tc-edit">
-<<lingo Fields/Add/Prompt>>&nbsp;
+<<lingo Fields/Add/Prompt>>&nbsp;&nbsp;
```

Comment:
as I wrote ... I'd prefer a CSS solution

Reply:
I think this is a reasonable quick fix.